jaq-core/src/lib.rs

```diff
@@ -402,3 +404,93 @@ const LOG: &[(&str, usize, RunPtr, UpdatePtr)] = &[(
     |_, cv| box_once(Ok(debug(cv.1))),
     |_, cv, f| f(debug(cv.1)),
 )];
+
+#[cfg(feature = "format")]
+fn to_csv(v: Val, delimiter: u8) -> ValR {
```

Comment:
Here, it would be preferable to accept a `&[Val]` and to call this function on `v.as_str()`.

Reply:
Done!